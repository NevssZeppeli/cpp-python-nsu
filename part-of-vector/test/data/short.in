28
62
70
27
53
65
-13
53
7
55
34
32
99
27
-18
33
-1
97
49
5
55
-16
36
-11
44
63
82
-18
63
13
46
56
54
57
-1
18
-19
-12
-2
46
-1
-11
58
96
23
67
97
52
77
47
