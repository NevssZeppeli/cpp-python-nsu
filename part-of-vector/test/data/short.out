65 53 27 70 62 28